nautobot/extras/jobs.py

```diff
@@ -189,6 +191,25 @@ def before_start(self, task_id, args, kwargs):
             )
             raise RunJobTaskFailed(f"Job {self.job_model} is not enabled to be run!")
 
+        if self.job_model.is_singleton:
+            if kwargs.get("_force_singleton_lock"):
+                cache.delete(self.singleton_cache_key)
```

Comment:
Feels off to me that this clears/resets the cache key. Why not just bypass the `is_running` check in this case instead?